### DISCUSSION TITLE
How to control subsetting behavior when using `Materialize all`?

### DISCUSSION THREAD

**Discussion by dduong1603** [dagster-io/dagster#18370] [Q&A] [answered]
I encountered the error `OSError: [Errno 7] Argument list too long: 'dbt'` when trying to materialize all of my dbt assets at the same time. This happens on k8s and not my local mac so I think it's just a limitation on Linux. Is there a workaround for this limitation, or a way to control the subsetting behavior when trying to materialize all/a large number of assets?

**Comment by rexledesma** [comment, marked as answer]
This is a known issue: https://github.com/dagster-io/dagster/issues/16997. I'm closing this discussion in favor on consolidating on that issue.

**Reply by dduong1603** [reply]
Oh ok thanks @rexledesma. Will add the details on that issue
